src/env.cc

```diff
@@ -1102,9 +1104,9 @@ void AsyncHooks::Deserialize(Local<Context> context) {
   native_execution_async_resources_.resize(
       info_->native_execution_async_resources.size());
   for (size_t i = 0; i < info_->native_execution_async_resources.size(); ++i) {
-    v8::Local<v8::Object> obj =
+    Local<Object> obj =
         context
-            ->GetDataFromSnapshotOnce<v8::Object>(
+            ->GetDataFromSnapshotOnce<Object>(
```

Comment:
This looks a little ugly now too, can't submit a code suggestion because I'm not sure what the style guides are regarding this 🤔

Reply:
Just did the same as suggested above :)